### ISSUE TITLE
Run tests for SCS-compatible IaaS via Zuul

### ISSUE THREAD

**Issue by mbuechse** [SovereignCloudStack/standards#316]
Build a prototype implementation of a Zuul CI job that runs the certification conformance tests. (Arose out of #268)

**Comment by mbuechse** [comment]
This seems to be a duplicate of https://github.com/SovereignCloudStack/issues/issues/279
Maybe we can rededicate this issue a bit and use it to provide the necessary project environment for the compliance checker (network, router etc.)

**Comment by mbuechse** [comment]
I guess we will get on with this after https://github.com/SovereignCloudStack/issues/issues/354

**Comment by fkr** [comment]
Currently blocked by Zuul not being available in a reliable fashion.

**Comment by fkr** [comment]
Zuul is back operational and the @mbuechse is currently waiting for the next run (to be finished in five minutes) that will emit a first json results file for gx-scs.
